ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDialog.kt

```diff
@@ -69,6 +81,21 @@ private fun DialogScaffold(paywallOptions: PaywallOptions) {
     }
 }
 
+@Composable
+@ReadOnlyComposable
+private fun getDialogMaxHeightPercentage(): Float {
+    val orientation = LocalConfiguration.current.orientation
+    if (orientation == Configuration.ORIENTATION_LANDSCAPE) {
+        return 1f
+    }
```

Comment:
I see what you're trying to do here. I wonder if a better approach to this to also work with multi-tasking (side by side apps, whatever Android calls it) is to check the application window aspect ratio?

Though I guess even on that case this implementation makes sense.

Reply:
Right, that's a good point... With the current implementation, when using multitasking, if the view is mostly square, the dialog won't use the full height, but it would look better if it did. Will change that 👍

Reply:
After the change:
![image](https://github.com/RevenueCat/purchases-android/assets/808417/18a2dfd2-4247-4664-b780-e08cbb0367b0)